fivetran_blueprints/check_sync_status.py

```diff
@@ -138,14 +136,13 @@ def main():
             if full_pickle_path:
```

Comment:
Also, won't this just result in a loop where the last item in the list is the one used for loading the pickle variables?

I _think_ that's dangerous because you could have multiple upstream Vessels where only 1 out of 3 is actually the correct Fivetran pickle file.